.dxscannerrc.yaml

```diff
@@ -1,2 +1,2 @@
 practices:
-  JavaScript.DependenciesVersionMinorPatchLevel: medium
+  JavaScript.DependenciesVersionMinorPatchLevel: medium
```

Comment:
```suggestion
  JavaScript.DependenciesVersionMinorPatchLevel: medium
  
```

Reply:
@adelkahomolova thank you for the review, unfortunately I don't understand German. Maybe @prokopsimek can assist me?

Reply:
@ryzzaki Me neither. 🤪